go/src/lib.rs

```diff
@@ -1,9 +1,159 @@
+#![deny(unsafe_op_in_unsafe_fn)]
```

Comment:
Move this after license header

Reply:
The Rust code exists in this PR to get tests passing but we are reviewing it under a separate PR here: https://github.com/aws/glide-for-redis/pull/1127/files